docs/REST_API.md

```diff
@@ -230,6 +233,12 @@ GET /config
       "engine": "ScanPy version 1.33",
       "dataset": "/home/joe/mouse/blorth.csv"
     },
+    "parameters": {
+      "verbose": false,
+      "log-level": "info",
```

Comment:
Where are verbose and log-level coming from? Is this the `--debug` param? If so, what are our log-levels? We should probably define them.

Reply:
they are just examples - intended to show what _might_ be.

you are right that when we add our first parameter, we need to document it.  Probably need to add a section that describes  each.   Given that we do not yet have any, I made up some examples.

Reply:
BTW, we could hold this PR (and add to it) once we have our first set of CLI params defined.

Reply:
Yeah, let's do that. The doc looks good outside of the example params. Technically they are fine, but I'm worried they would be confusing to a new contributor who thinks they are required.